Author: Andreas Cord-Landwehr<cola@uni-paderborn.de>
Author: Kevin Kofler<kevin.kofler@chello.at>
Description: Build Rocs by default with Boost Exceptions.
 This solves the common build issues with many Boost releases.
Applied-Upstream: 4.8, commit:43b6ccf22174629ebec0a0579365c4269cf0fb58
Applied-Upstream: 4.8, commit:a2f525ffd6f28a635ba3231734bdba9747c8ef29

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,6 +8,8 @@ find_package(KDE4 REQUIRED)
 include(KDE4Defaults)
 remove_definitions(-DQT_NO_STL)
 
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${KDE4_ENABLE_EXCEPTIONS}" )
+
 include_directories(
   ${KDE4_INCLUDES}
   ${QT_INCLUDES}
@@ -15,7 +17,6 @@ include_directories(
 
 if(${CMAKE_BUILD_TYPE} MATCHES coverage)
   set (CMAKE_CXX_COMPILER cov++)
-  set (CMAKE_CXX_FLAGS "${KDE4_ENABLE_EXCEPTIONS}" )
 endif()
 
 find_package(Boost REQUIRED)
